controller/src/http_server.rs

```diff
@@ -68,9 +68,12 @@ impl HttpSpawnRequest {
 
         let lock: Option<ResourceLock> = self.lock.clone().map(|l| l.try_into()).transpose()?;
 
+        let mut env = self.env.clone();
+        env.insert("PORT".to_string(), "8080".to_string());
```

Comment:
Shouldn't this be `self.port` and if that's not defined, 8080? As written now, if I spawn a backend with port 4000, PORT environment variable will be set to 8080

Reply:
Good catch.